src/Control/Monad/Eff/JQuery.purs

```diff
@@ -63,6 +118,24 @@ foreign import setAttr
   -> JQuery
   -> Eff (dom :: DOM | eff) Unit
 
+foreign import getAttrImpl
+  :: forall eff
+   . String
+  -> JQuery
+  -> Eff (dom :: DOM | eff) Foreign
+
+-- | Get an attribute value.
+getAttr 
+  :: forall eff
+   . String
+   -> JQuery
+   -> Eff (dom :: DOM | eff) (Maybe String)
+getAttr str jq = map foreignToString (getAttrImpl str jq)
+  where foreignToString f =
+          if isUndefined f
+            then Nothing
+            else Just (unsafeFromForeign f)
```

Comment:
not evaluated, the `$` throws error. `select :: forall eff. Selector -> Eff (dom :: DOM | eff) JQuery` doesn't have `err | Exception` but it does throw exceptions.
The proposition `forall eff. String -> JQuery -> Eff (dom :: DOM | eff) (Maybe String)` is correct.